### PR TITLE
fixed issues with enable reward points

### PR DESCRIPTION
**Pull request by joshua1234511**

### CODE REVIEW

File: application/models/Sale.php [hunk 1]
```diff
@@ -536,7 +536,7 @@ public function save($items, $customer_id, $employee_id, $comment, $invoice_numb
 			$total_amount = floatval($total_amount) + floatval($payment['payment_amount']);
 		}
 
-		if(isset($customer_id) && $customer_id!= NULL){
+		if(isset($customer_id) && $customer_id!= NULL && $this->config->item('customer_reward_enable') == TRUE){
```
Comment (daN4cat) [inline application/models/Sale.php]: you could have used if(!empty (-$customer_id))

Comment (joshua1234511) [inline application/models/Sale.php]: used and commited

File: application/models/Sale.php [hunk 2]
```diff
@@ -536,7 +536,7 @@ public function save($items, $customer_id, $employee_id, $comment, $invoice_numb
 			$total_amount = floatval($total_amount) + floatval($payment['payment_amount']);
 		}
 
-		if(isset($customer_id) && $customer_id!= NULL){
+		if((!empty (-$customer_id)) && $this->config->item('customer_reward_enable') == TRUE){
```
Comment (daN4cat) [inline application/models/Sale.php]: is the - on purpose? Looks a bit odd to me

File: application/models/Sale.php [hunk 3]
```diff
@@ -536,7 +536,7 @@ public function save($items, $customer_id, $employee_id, $comment, $invoice_numb
...
 			$package_id = $this->Customer->get_info($customer_id)->package_id;
 				if(isset($package_id) && $package_id!=NULL){
```
Comment (daN4cat) [inline application/models/Sale.php]: You could use !empty even here

File: application/models/Sale.php (hunk 2 above)
Comment (daN4cat) [inline application/models/Sale.php]: hmmm ok it was a typo on my side in my previous comment.

**Comment by daN4cat** [comment]
I'm fixing the typo and commit the change to master.

**Comment by joshua1234511** [comment]
okay thanks :)

**Pull request merged**